lib/cuke_cataloger/unique_test_case_tagger.rb

```diff
@@ -398,19 +398,14 @@ def update_rows_if_needed(test, sub_id, id_column_name)
     end
 
 
-    # Slowest way to get the id tag. Will check the object every time.
-    def current_id_tag_for(thing)
-      id_tag_for(thing)
-    end
-
```

Comment:
Although the comment on the method says that it is the slowest of the three tag getting methods, I would like to be able to guess that from the method name as well. So I would rather get rid of `#id_tag_for` and move it's implementation into this method rather than get rid of `#current_id_tag_for`. Or we could just add it as an alias, I suppose.